### PR TITLE
Make sponsoring(challenges & km-sponsoring) easier 

### PR DESCRIPTION
**Pull request by MkuuWaUjinga**
This PR makes it possible to create a sponsoring directly on a team page.

Requires: 
https://github.com/BreakOutEvent/breakout-api-client v0.14.0
https://github.com/BreakOutEvent/breakout-backend/pull/231

### CODE REVIEW

File: src/client/react/App.jsx [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+import {
+  BrowserRouter as Router,
+  Route,
+  Redirect
+} from 'react-router-dom';
+import PropTypes from 'prop-types';
+import i18next from 'i18next';
+import MuiThemeProvider from '@material-ui/core/styles/MuiThemeProvider';
+import { createMuiTheme } from '@material-ui/core/styles';
+import orange from '@material-ui/core/colors/red';
```
Comment (nerdsupremacist) [inline src/client/react/App.jsx]: `orange = red`?

File: src/client/react/App.jsx [hunk 2]
```diff
@@ -283,7 +303,7 @@ function renderIfExists(elem, domId) {
   if (domNode) {
     ReactDOM.render(elem, domNode);
   } else {
-    console.warn(`Not rendering react component because node with id ${domId} does not exist`);
+    // console.warn(`Not rendering react component because node with id ${domId} does not exist`);
```
Comment (nerdsupremacist) [inline src/client/react/App.jsx]: why is this commented out?

Comment (dominikglandorf) [reply src/client/react/App.jsx]: Because it is expected behaviour and no error.

File: src/client/react/App.jsx (hunk 1 above)
Comment (dominikglandorf) [reply src/client/react/App.jsx]: Thx!

File: src/client/react/App.jsx (hunk 2 above)
Comment (florianschmidt1994) [reply src/client/react/App.jsx]: I've been thinking about this a while when I implemented it. IMHO it should be a debug log

**Review by nerdsupremacist — APPROVED** [review]

**Pull request merged**